### ISSUE TITLE
[marketing] Concise demo of DXOS that can be reused in-person and online

### ISSUE THREAD

**Issue by zhenyasav** [dxos/dxos#3431]
## Motivation

We a script that we can follow for demoing along with the associated resources to execute on the script (apps, features, etc) that convey the main benefits and "how it works" of DXOS without overwhelming a developer but piquing their interest.

## Finish Line

- Demo script in a document in Composer so that others can lean on it for their demos if they wish
- ~~Recording of the demo being delivered with voiceovers so that it can be shared async~~

## Out of Scope

- editing and polishing the video to make it more professional
- some of the more advanced features like agents, plugins, etc

**Comment by jessmartin** [comment]
https://composer.dxos.org/dxos/space/040d2150/56d858d09e636839fdc5193d1c605ce102a7af9627698f75495978f7a65c139b

**Comment by jessmartin** [comment]
@zhenyasav can you review and see if you'd like any more detail in the script? It's pretty loose, but I improvise a lot and so didn't want to over-specify.

**Comment by jessmartin** [comment]
waiting on the design updates to the invite flow to record a video
